bin/build-release-ccd-definition.sh

```diff
@@ -6,9 +6,9 @@ environment=${1:-prod}
 excludeNonProdFiles=${2:-true}
 
 if [ ${environment} == preview ]; then
-   excludedFilenamePatterns="-e *-prod.json,*HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*CUI-prod.json"
+   #excludedFilenamePatterns="-e *-prod.json,*HNL-nonprod.json,*CUI.json,*CUI-nonprod.json,*CUI-prod.json"
    #for testing HNL uplifting, comment the above line and uncomment below
-   #excludedFilenamePatterns="-e *-prod.json,*-SDO-nonprod.json,*-base-nonprod.json"
+   excludedFilenamePatterns="-e *-prod.json,*-SDO-nonprod.json,*-base-nonprod.json"
```

Comment:
These changes will need to be reverted before merging

Reply:
This has been reverted